packages/imperative/package.json

```diff
@@ -30,9 +30,9 @@
   "scripts": {
     "build:packages": "tsc --pretty",
     "build": "npm run build:packages && npm run build:webHelp",
-    "lint": "eslint \"packages/**/*.ts\" \"**/__tests__/**/*.ts\"",
-    "lint:packages": "eslint \"packages/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
-    "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
+    "lint": "eslint \"src/**/*.ts\" \"**/__tests__/**/*.ts\"",
+    "lint:packages": "eslint \"src/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
+    "lint:tests": "eslint \"./__tests__/**/*.ts\"",
```

Comment:
Don't we need to keep the `**` to include all tests because there are some inside subdirectories such as `packages/imperative/logger/__tests__`?